OpenChange/MAPIStoreAppointmentWrapper.m

```diff
@@ -552,9 +558,7 @@ - (int) getPidTagOwnerAppointmentId: (void **) data
 - (int) getPidLidMeetingType: (void **) data
                     inMemCtx: (TALLOC_CTX *) memCtx
 {
-  /* TODO
-     See 2.2.6.5 PidLidMeetingType (OXOCAL) */
-  *data = MAPILongValue (memCtx, 0x00000001);
+  *data = MAPILongValue (memCtx, 0x00000000);
```

Comment:
Why don't you use here the constant you defined in https://github.com/zentyal/openchange/pull/204?